Core/include/Acts/Utilities/QuickMath.hpp

```diff
@@ -15,12 +15,12 @@
 
 namespace Acts {
 
-/// @brief Fast inverse square root function
+/// @brief Quick inverse square root function
 /// Taken from https://en.wikipedia.org/wiki/Fast_inverse_square_root
 /// @param x the number to calculate the inverse square root of
 /// @param iterations the number of newton iterations to perform
 template <typename T>
-constexpr T fastInverseSqrt(T x, int iterations = 1) noexcept {
+constexpr T quickInverseSqrt(T x, int iterations = 1) noexcept {
```

Comment:
I know this is not covered in this PR, but my understanding is that, in modern compilers, this Quake (?) square root function isn't actually any faster than the `sqrt` builtin.

Reply:
https://github.com/acts-project/acts/pull/3701